examples/somersaultecu.py

```diff
@@ -268,6 +269,7 @@ class SomersaultSID(IntEnum):
             short_name="session",
             long_name="Session",
             description=None,
+            admin_data=None,
```

Comment:
This is why I like trailing comma

Reply:
no particular reason. probably because it makes it possible to paste this line at any given position (odxtools is inconsistent when it comes to this...)